templates/repo/view_list.tmpl

```diff
@@ -1,4 +1,4 @@
-<table id="repo-files-table" class="ui single line table tw-mt-0" {{if .HasFilesWithoutLatestCommit}}hx-indicator="tr.notready td.message span" hx-trigger="load" hx-swap="morph" hx-post="{{.LastCommitLoaderURL}}"{{end}}>
+<table id="repo-files-table" class="ui single line fixed table tw-mt-0" {{if .HasFilesWithoutLatestCommit}}hx-indicator="tr.notready td.message span" hx-trigger="load" hx-swap="morph" hx-post="{{.LastCommitLoaderURL}}"{{end}}>
```

Comment:
This `fixed` causes commit messages to cut off too soon and the timestamp column to get too wide. But without it, they table overflows on certain viewports. It needs more investigation.

Reply:
https://github.com/go-gitea/gitea/pull/32759 will fix that and revert the table layout.